include/LVSS.hpp

```diff
@@ -1,13 +1,59 @@
-#pragma once
+#ifndef _LVSS_
+#define _LVSS_
+
+#include "EVT/dev/LCD.hpp"
```

Comment:
Use angle brackets for all your includes
```suggestion
#include <EVT/dev/LCD.hpp>
```